src/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+    <title>Leaderboard</title>
+</head>
+<body>
+    <nav class="Container-fluid">
+        <div class="top-nav">
+            <div class="top-part">
+              <h2>A Web Page</h2>
+            </div>
+            <div class="bottom-part">
+              <i class="fa-solid fa-left-long fa-3x white"></i>
+              <i class="fa-solid fa-right-long white fa-3x"></i>
+              <i class="fa-solid fa-xmark fa-3x"></i>
+              <i class="fa-solid fa-house fa-2x white"></i>
+              <input type="text" placeholder="https://" class="http-input">
+              <div class="search-bar">
+              <input type='text' placeholder='Search...' id='search-text-input' class="curved-input"/><i class="fa-solid fa-magnifying-glass"></i>
+            </div>
+        </div>
+    </nav>
```

Comment:
[ Optional Change ]
This change won't stop your project from being approved but it would be nice if you consider working on it.

- [ ] This section on top is actually not needed in your app design. It's basically showing the browser's URL but from a Wireframe design perspective. On your real app design, you already have your own browser URL header and so does anyone who runs your app. Therefore, you could remove it and have the remaining code below it. 👍🏽

Reply:
Thanks a lot 👍